http/src/http.mli

```diff
@@ -21,6 +21,7 @@ module Method : sig
     | `Other of string ]
 
   val compare : t -> t -> int
+  val body_allowed : t -> bool
```

Comment:
```suggestion
  val body_allowed : t -> bool
  (** [body_allowed meth] is true if the http method allows a payload
      body to be present following RFC7231.  *)
  
```
I think we should document this method. This is just a suggestion, feel free to edit as appropriate